README.md

```diff
@@ -15,70 +15,77 @@
 [s6]: https://tokei.rs/b1/github/amethyst/laminar?category=code
 [s7]: https://codecov.io/gh/amethyst/laminar/branch/master/graphs/badge.svg
 
-A UDP-based protocol that provides partial reliability. Coming soon!
+This library implements some TCP-like features on top of an UDP-socket. 
+I will provide a lightweight message-based interface with certain guarantees like reliability, fragmentation congestion monitoring.
 
-## Note
+Laminar was designed to be used in the [Amethyst][amethyst] game engine and is mostly based on articles from [gaffer on games](https://gafferongames.com/).
```

Comment:
I would change mostly to loosely, since we used other resources and our own brains too.

Reply:
true